client/views/room/hooks/useUserInfoActions.js

```diff
@@ -299,9 +299,10 @@ export const useUserInfoActions = (user = {}, rid) => {
 				userId={user?._id}
 				onClose={closeModal}
 				onCancel={closeModal}
-				onConfirm={(rooms) => {
-					removeFromTeam({ teamId: room.teamId, members: [{ userId: user._id }], rooms });
+				onConfirm={async (rooms) => {
```

Comment:
@ggazzo IDK but... there's a better approach to use here instead of the async function?